test/attr/Inputs/DeprecatedSymbolsModule.swift

```diff
@@ -0,0 +1,20 @@
+@available(*, deprecated)
+func importedDeprecatedFunc() {}
+
+@available(*, deprecated)
+func importedDeprecatedFuncWithReturn() -> String {
+    return "Donda"
+}
+
+@available(*, deprecated)
+var importedDeprecatedVar = "910210"
+
+@available(*, deprecated)
+struct importedDeprecatedType {
+    var x = ""
+    static let m = 22
+
+    func memberFunc() -> Int {
+        40
+    }
+}
```

Comment:
These will all need to be marked `public` to be accessible to the client test file